[18-Feb-2020 05:23:04 Europe/Berlin] Uncaught Exception: 'PDOException'
Message : 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'urlKey''
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Admins.php(13): PDO->exec('INSERT INTO pro...')
#1 C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Admin\Product.php(20): App\Models\Admins::insertData(Array, 'products')
#2 [internal function]: App\Controllers\Admin\Product->addAction()
#3 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(44): Core\Controller->__call('add', Array)
#5 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(17): Core\Router->dispatch('admin/Product/a...')
#6 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Admins.php' on Line13
[18-Feb-2020 05:24:53 Europe/Berlin] Uncaught Exception: 'PDOException'
Message : 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'urlKey''
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Admins.php(12): PDO->exec('INSERT INTO pro...')
#1 C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Admin\Product.php(20): App\Models\Admins::insertData(Array, 'products')
#2 [internal function]: App\Controllers\Admin\Product->addAction()
#3 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(44): Core\Controller->__call('add', Array)
#5 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(17): Core\Router->dispatch('admin/Product/a...')
#6 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Admins.php' on Line12
[18-Feb-2020 05:25:48 Europe/Berlin] Uncaught Exception: 'PDOException'
Message : 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'urlKey''
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Admins.php(12): PDO->exec('INSERT INTO pro...')
#1 C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Admin\Product.php(20): App\Models\Admins::insertData(Array, 'products')
#2 [internal function]: App\Controllers\Admin\Product->addAction()
#3 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(44): Core\Controller->__call('add', Array)
#5 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(17): Core\Router->dispatch('admin/Product/a...')
#6 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Admins.php' on Line12
[18-Feb-2020 05:26:15 Europe/Berlin] Uncaught Exception: 'PDOException'
Message : 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'urlKey''
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Admins.php(12): PDO->exec('INSERT INTO pro...')
#1 C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Admin\Product.php(20): App\Models\Admins::insertData(Array, 'products')
#2 [internal function]: App\Controllers\Admin\Product->addAction()
#3 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(44): Core\Controller->__call('add', Array)
#5 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(17): Core\Router->dispatch('admin/Product/a...')
#6 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Admins.php' on Line12
[18-Feb-2020 08:48:02 Europe/Berlin] Uncaught Exception: 'Exception'
Message : 'No Route matched'
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(17): Core\Router->dispatch('category/view/')
#1 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php' on Line55
[18-Feb-2020 09:27:32 Europe/Berlin] Uncaught Exception: 'Exception'
Message : 'No Route matched'
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(17): Core\Router->dispatch('admin/Product/')
#1 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php' on Line56
[18-Feb-2020 09:27:37 Europe/Berlin] Uncaught Exception: 'Exception'
Message : 'No Route matched'
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(17): Core\Router->dispatch('admin/Product/e...')
#1 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php' on Line56
[18-Feb-2020 09:28:36 Europe/Berlin] Uncaught Exception: 'Exception'
Message : 'No Route matched'
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(17): Core\Router->dispatch('admin/Product/e...')
#1 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php' on Line56
[18-Feb-2020 10:17:30 Europe/Berlin] Uncaught Exception: 'Exception'
Message : 'Method viewAction not found in controllerApp\Controllers\Admin\Product'
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(49): Core\Controller->__call('view', Array)
#1 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(19): Core\Router->dispatch('admin/product/v...')
#2 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php' on Line18
[18-Feb-2020 10:52:50 Europe/Berlin] Uncaught Exception: 'Exception'
Message : 'No Page Found'
Stack Trace :#0 [internal function]: App\Controllers\CmsPage->viewAction('aboutuss')
#1 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(46): Core\Controller->__call('view', Array)
#3 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('cmsPage/view/ab...')
#4 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\cmsPage.php' on Line19
[18-Feb-2020 10:57:42 Europe/Berlin] Uncaught Exception: 'Exception'
Message : 'No Page Found'
Stack Trace :#0 [internal function]: App\Controllers\CmsPage->viewAction('home12')
#1 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(46): Core\Controller->__call('view', Array)
#3 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('cmsPage/view/ho...')
#4 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\cmsPage.php' on Line19
[18-Feb-2020 10:59:07 Europe/Berlin] Uncaught Exception: 'Exception'
Message : 'No Route matched'
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('admin/Category/...')
#1 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php' on Line61
[18-Feb-2020 11:06:29 Europe/Berlin] Uncaught Exception: 'Error'
Message : 'Call to undefined method App\Models\Categories::fetchParents()'
Stack Trace :#0 [internal function]: App\Controllers\Home->indexAction()
#1 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(49): Core\Controller->__call('index', Array)
#3 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('')
#4 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Home.php' on Line13
[18-Feb-2020 11:06:35 Europe/Berlin] Uncaught Exception: 'Error'
Message : 'Call to undefined method App\Models\Categories::fetchParents()'
Stack Trace :#0 [internal function]: App\Controllers\Home->indexAction()
#1 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(49): Core\Controller->__call('index', Array)
#3 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('')
#4 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Home.php' on Line13
[18-Feb-2020 11:06:41 Europe/Berlin] Uncaught Exception: 'Error'
Message : 'Call to undefined method App\Models\Categories::fetchParents()'
Stack Trace :#0 [internal function]: App\Controllers\Home->indexAction()
#1 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(49): Core\Controller->__call('index', Array)
#3 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('')
#4 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Home.php' on Line13
[18-Feb-2020 11:06:47 Europe/Berlin] Uncaught Exception: 'Error'
Message : 'Call to undefined method App\Models\Categories::fetchParents()'
Stack Trace :#0 [internal function]: App\Controllers\Home->indexAction()
#1 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(49): Core\Controller->__call('index', Array)
#3 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('')
#4 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Home.php' on Line13
[18-Feb-2020 11:06:52 Europe/Berlin] Uncaught Exception: 'Error'
Message : 'Call to undefined method App\Models\Categories::fetchParents()'
Stack Trace :#0 [internal function]: App\Controllers\Product->viewAction('apple')
#1 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(46): Core\Controller->__call('view', Array)
#3 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('product/view/ap...')
#4 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Product.php' on Line9
[18-Feb-2020 11:12:08 Europe/Berlin] Uncaught Exception: 'ErrorException'
Message : 'Object of class PDOStatement could not be converted to string'
Stack Trace :#0 C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Categories.php(8): Core\Error::errorHandler(4096, 'Object of class...', 'C:\\xampp\\htdocs...', 8, Array)
#1 C:\xampp\htdocs\MVCFRAMEWORK\App\Controllers\Category.php(12): App\Models\Categories::displayData('mobile')
#2 [internal function]: App\Controllers\Category->viewAction('mobile')
#3 C:\xampp\htdocs\MVCFRAMEWORK\Core\Controller.php(13): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\MVCFRAMEWORK\Core\Router.php(46): Core\Controller->__call('view', Array)
#5 C:\xampp\htdocs\MVCFRAMEWORK\public\index.php(21): Core\Router->dispatch('category/view/m...')
#6 {main}
Thrown in 'C:\xampp\htdocs\MVCFRAMEWORK\App\Models\Categories.php' on Line8
